scrape.md

```diff
@@ -57,13 +57,13 @@ artoo.scrape('li', {id: 'id', content: 'text'});
   * **done**    *?function* : same as `callback`.
 * **callback**  *?function* : a function taking as only argument the scraped items and to be triggered when the scraping is done.
 
-Alternatively, you can pass a single object as argument to the `scrape` method and taking as properties `iterator`, `data` and `params`.
+Alternatively, you can pass a single object as an argument to the `scrape` method and setting the properties `iterator`, `data` and `params`.
 
 ---
 
 ### Data model
 
-Choosing a data model when using `artoo.scrape` is just a matter of deciding whether you want the function to return an array of values or rather an array of items with properties you prealably chose.
+Choosing a data model when using `artoo.scrape` is just a matter of deciding whether you want the function to return an array of values or rather an object with items and properties you designate.
```

Comment:
Yes, it does change the meaning.  My understanding of the way artoo works matches closer to designate.  But you are the creator, so we will go with what makes sense to you.

Either way "prealably" is not correct.  Perhaps "preferably"?  But then it would be "preferably chose"  which is redundent.  So maybe " with properties you choose"